Solutions/Legacy IOC based Threat Protection/Deprecated Analytic Rules/CadetBlizzard_Jan2022_IOC.yaml

```diff
@@ -72,27 +72,38 @@ query:  |
   | where (Hashes[0] =~ "SHA256" and Hashes[1] has_any (sha256Hashes)) or ( CommandLine has ('127.0.0.1\\ADMIN$') and CommandLine has_any (Command_Line)) 
   | project TimeGenerated, EventDetail, UserName, Computer, Type, Source, Hashes, CommandLine, Image
   | extend Type = strcat(Type, ": ", Source)
-  | extend timestamp = TimeGenerated, HostCustomEntity = Computer , AccountCustomEntity = UserName, ProcessCustomEntity = tostring(split(Image, '\\', -1)[-1]), FileHashCustomEntity = tostring(Hashes[1]), AlgorithmCustomEntity = "SHA256"
+  | DvcHostName = Computer , ActorUsername = UserName, ProcessName = tostring(split(Image, '\\', -1)[-1]), FileHashName = tostring(Hashes[1]), AlgorithmName = "SHA256"
```

Comment:
Missed ```| extend``` at the beginning and ProcessName is not ProcessId as indicated in the other query, this will need to be changed to the ProcessId field coming from this log.